docs/user/BuildLocally.md

```diff
@@ -16,7 +16,8 @@ sudo ./setup.sh
 ## Build
 
 ``` shell
-./build_openroad.sh --local
+sudo apt-get install pandoc
```

Comment:
This should be handled by the dependency installer rather than making it a manual step.  @vvbandeira

Reply:
I will propose a PR, passing `--openroad-args " -DBUILD_MAN=ON"` should not also be needed.